src/plugins/supply-demand/languages/BasicJS/starter-code/main.js

```diff
@@ -1,18 +1,30 @@
-// Base curves are created automatically when you use shift functions!
+//law of supply demo
```

Comment:
nit: Make sure your spacing/indenting is correct here -- run prettier on these files?